test_pyganja.py

```diff
@@ -42,6 +42,49 @@ def test_draw_points(self):
         render_cef_script(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=False)
 
 
+class TestPGA3DDrawing(unittest.TestCase):
+    def test_draw_points(self):
+        from clifford.pga import layout, blades
+        import numpy as np
+        def random_point():
+            return sum([c*b for c,b in zip(np.random.randn(3), [blades["e023"], blades["e013"], blades["e012"]])]) + blades["e123"]
+        gs = GanjaScene()
+        gs.add_objects([random_point() for i in range(10)], static=False)
+        with open('test_file.html', 'w') as test_file:
+            print(generate_full_html(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=True), file=test_file)
```

Comment:
`print` adds extra newlines, you maybe want just:
```suggestion
            f.write(generate_full_html(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=True))
```